File: system-test/sec-tests/javascript/sec_58_brute_force.js
// SEC-58: Brute Force Protection Missing
// Test: TC-CR-061
app.post('/login', (req, res) => {
  const { username, password } = req.body;
  // No rate limiting, no account lockout
  if (authenticate(username, password)) {
    res.send('Login success');
  } else {
    res.send('Login failed');
  }
});

File: system-test/sec-tests/python/sec_58_brute_force.py
# SEC-58: Brute Force Protection Missing
# Test: TC-CR-061
from flask import Flask, request

app = Flask(__name__)

@app.route('/login', methods=['POST'])
def login():
    username = request.form['username']
    password = request.form['password']
    # No rate limiting, no account lockout
    if check_password(username, password):
        return 'login success'
    return 'login failed'
